src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage.jsx

```diff
@@ -0,0 +1,224 @@
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import { Link, useHistory, useParams } from 'react-router-dom';
+import LoadingIndicator from '@department-of-veterans-affairs/component-library/LoadingIndicator';
+
+import AddToCalendar from '../../components/AddToCalendar';
+import VAFacilityLocation from '../../components/VAFacilityLocation';
+import moment from '../../lib/moment-tz';
+import {
+  getVAAppointmentLocationId,
+  getVARFacilityId,
+  isVideoAppointment,
+} from '../../services/appointment';
+import { FETCH_STATUS, PURPOSE_TEXT } from '../../utils/constants';
+import { scrollAndFocus } from '../../utils/scrollAndFocus';
+import * as actions from '../redux/actions';
+import { cancelAppointment } from '../redux/actions';
+import AppointmentDateTime from './cards/confirmed/AppointmentDateTime';
+import AppointmentInstructions from './cards/confirmed/AppointmentInstructions';
+
+// Only use this when we need to pass data that comes back from one of our
+// services files to one of the older api functions
+function parseFakeFHIRId(id) {
+  return id ? id.replace('var', '') : id;
+}
+
+function formatAppointmentDate(date) {
+  if (!date.isValid()) {
+    return null;
+  }
+
+  return date.format('MMMM D, YYYY');
+}
+
+function ConfirmedAppointmentDetailsPage({
+  appointmentDetails,
+  appointmentDetailsStatus,
+  facilityData,
+  fetchConfirmedAppointmentDetails,
+  confirmedStatus,
+  showCancelButton,
+}) {
+  const { id } = useParams();
+  const history = useHistory();
+
+  const appointment = appointmentDetails?.[id];
+
+  useEffect(() => {
+    const status = confirmedStatus === FETCH_STATUS.succeeded;
+
+    if (!status) {
+      history.push('/va');
+    }
+
+    if (!appointment) {
+      fetchConfirmedAppointmentDetails(id);
+    }
+
+    scrollAndFocus();
+  }, []);
+
+  if (appointmentDetailsStatus === FETCH_STATUS.loading) {
+    return (
+      <div className="vads-u-margin-y--8">
+        <LoadingIndicator message="Loading your appointment request...!!!!" />
+      </div>
+    );
+  }
+
+  if (!appointment) {
+    return null;
+  }
+
+  const facilityId = getVAAppointmentLocationId(appointment);
+  const facility = facilityData?.[facilityId];
+
+  const isCC = appointment?.vaos.isCommunityCare;
+  const isExpressCare = appointment?.vaos.isExpressCare;
+  const isCommunityCare = appointment?.vaos.isCommunityCare;
```

Comment:
Community care is kind of tricky here, since the majority of CC appointments won't use this page, they get data from a different service. But we do have the potential for CC appointments from the VA appointments service (it's turned off in prod, though). 

I'm not positive about the best way to handle this, but I would lean toward assuming `isCommunityCare` is always false here and handling both types of CC appointments in the CC appointment details page.

Reply:
I'm not sure I understand completely but does creating another service or modifying the current getBookedAppointments service (flag or something) to only get VA confirmed appointments work?

Reply:
I don't think you need to modify anything in the services, I think that you can assume that this page will never need to render an appointment that's community care. We can make sure the list pages link to a different page than this one when the appointment is CC.

Reply:
Ok